api/tests/integration/infrastructure/repositories/tutorial-repository_test.js

```diff
@@ -139,7 +139,7 @@ describe('Integration | Repository | tutorial-repository', function () {
     });
 
     context('when user has saved tutorials', function () {
-      it('should return tutorial with user tutorial belonging to given user ordered by userTutorial Id desc', async function () {
+      it('should return tutorial with user tutorial belonging to given user ordered by userTutorial createdAt desc', async function () {
```

Comment:
```suggestion
it('should return tutorial with user tutorial belonging to given user ordered by descending date of creation', async function () {
```

**Suggestion :** Pour lier un peu moins à l'implémentation